### PR TITLE
#103 install xml2rfc deps

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #103

### CODE REVIEW

**Comment by CAMOBAP** [comment]
As I understand the validation scheme was changed, investigation in progress

**Comment by CAMOBAP** [comment]
@opoudjis I'm not 100% sure about removing this class variable could you please review

**Comment by opoudjis** [comment]
The `@done_rfc` is there to ensure that the File.exists has been recompiled; if it's been sitting around from an earlier compilation, it's not valid. Yes, please keep both.

**Review by opoudjis — CHANGES_REQUESTED** [review]
Please use both `@done_rfc` and File.exists

**Comment by CAMOBAP** [comment]
@opoudjis done

**Comment by CAMOBAP** [comment]
@ronaldtse @opoudjis could you please review again

File: lib/metanorma/ietf/processor.rb [hunk 1]
```diff
@@ -58,42 +58,31 @@ def output(isodoc_node, inname, outname, format, options={})
...
+            when :html then ".html"
           end
 
-          rfcname = inname.sub(/\.xml$/, ".rfc.xml")
-          output(isodoc_node, inname, rfcname, :rfc, options) unless @done_rfc
-
-          outname ||= inname.sub(/\.xml$/, ".html")
-          system("xml2rfc --html #{rfcname} -o #{outname}")
+          outflag = case format
+            when :txt then "--text"
```
Comment (ronaldtse) [inline lib/metanorma/ietf/processor.rb:79]: You can also write something like:

```ruby
outext, outflag = case format
  when :txt then [".txt", "--txt"]
  when :pdf then [".pdf", "--pdf"]
  when :html then [".html", "--html"]
end
```

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**